File: 2019 czerwiec/c++/1.cpp
#include <iostream>
#include <fstream>
using namespace std;

bool is_prime(int n)
{
    if (n <= 1)
        return false;
    for (int i = 2; i * i <= n; i++)
    {
        if (n % i == 0)
            return false;
    }
    return true;
}

int main()
{
    ifstream in("liczby.txt");
    ofstream out("wyniki4_1.txt");

    for (int i = 1; i <= 300; i++)
    {
        int a;
        in >> a;
        if (a >= 100 && a <= 5000 && is_prime(a))
            out << a << '\n';
    }
}

File: 2019 czerwiec/python/1.py
with open("liczby.txt", "r") as f:
    liczby = list(map(int, f.read().strip().split('\n')))

def is_prime(n):
    if n == 1:
        return False
    
    for i in range(2, n):
        if n % i == 0:
            return False
    return True

with open("wyniki4_1.txt", "w") as f:
    for n in liczby:
        if n >= 100 and n <= 5000 and is_prime(n):
            f.write(str(n) + '\n')